cms/static/js/spec/models/xblock_info_spec.js

```diff
@@ -7,16 +7,38 @@ define(['backbone', 'js/models/xblock_info'],
                 expect(new XBlockInfo({'category': 'sequential'}).isEditableOnCourseOutline()).toBe(true);
                 expect(new XBlockInfo({'category': 'vertical'}).isEditableOnCourseOutline()).toBe(true);
             });
+        });
+
+        describe('XblockInfo actions state and header visibility ', function() {
+
+            it('works correct to hide icons e.g. trash icon, drag when actions are not required', function(){
+                expect(new XBlockInfo({'category': 'chapter', 'actions': {'deletable':false}})
+                    .isDeletable()).toBe(false);
+                expect(new XBlockInfo({'category': 'chapter', 'actions': {'draggable':false}})
+                    .isDraggable()).toBe(false);
+                expect(new XBlockInfo({'category': 'chapter', 'actions': {'childAddable':false}})
+                    .isChildAddable()).toBe(false);
+            });
 
-            it('cannot delete an entrance exam', function(){
-                expect(new XBlockInfo({'category': 'chapter', 'override_type': {'is_entrance_exam':true}})
-                    .canBeDeleted()).toBe(false);
+            it('works correct to show icons e.g. trash icon, drag when actions are required', function(){
```

Comment:
Please also test the default behavior (when each of these are not specified, the helper "is" methods should still return true).